docs/src/sqlitegen/bubble-generator-data.tcl

```diff
@@ -457,6 +457,7 @@ set all_graphs {
           {line
               {opt dup}
               {opt datacopy}
+              {opt uniqnulls}
```

Comment:
We also need to re-generate the diagrams.

Reply:
It looks like I'll need to find a Linux machine with both X-Windows and Tk in order to do this.